kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/maps/homework/School.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.collections.adv.maps.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class School {
+    private String schoolName;
+    private List<Integer> students = new ArrayList();
+
+    public School(String schoolName, Integer... students) {
+        this.schoolName =schoolName;
+
+        for (Integer student : students) {
+            this.students.add(student);
+        }
+    }
+
+    public String getSchoolName() {
+        return schoolName;
+    }
+
+    public int getCountStudent() {
+        int sum = 0;
+        for(Integer student : students) {
+            sum += students.get(student);
```

Comment:
a tu co? Robisz petle for-each dla liczby uczniow. Czyli for (int student : students) { sum += student; }